presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/FileMetastoreModule.java

```diff
@@ -21,27 +21,18 @@
 import io.prestosql.plugin.hive.metastore.ExtendedHiveMetastore;
 
 import static io.airlift.configuration.ConfigBinder.configBinder;
-import static java.util.Objects.requireNonNull;
 import static org.weakref.jmx.ObjectNames.generatedNameOf;
 import static org.weakref.jmx.guice.ExportBinder.newExporter;
 
 public class FileMetastoreModule
         implements Module
 {
-    private final String connectorId;
-
-    public FileMetastoreModule(String connectorId)
-    {
-        this.connectorId = requireNonNull(connectorId, "connectorId is null");
-    }
-
     @Override
     public void configure(Binder binder)
     {
         configBinder(binder).bindConfig(FileHiveMetastoreConfig.class);
         binder.bind(ExtendedHiveMetastore.class).annotatedWith(ForCachingHiveMetastore.class).to(FileHiveMetastore.class).in(Scopes.SINGLETON);
         binder.bind(ExtendedHiveMetastore.class).to(CachingHiveMetastore.class).in(Scopes.SINGLETON);
-        newExporter(binder).export(ExtendedHiveMetastore.class)
-                .as(generatedNameOf(CachingHiveMetastore.class, connectorId));
+        newExporter(binder).export(ExtendedHiveMetastore.class).as(generator -> generatedNameOf(CachingHiveMetastore.class));
```

Comment:
This should be `generator.generatedNameOf`?

Reply:
Yes... the generator here is the default one inside of JmxUtils